src/components/Menu/Menu.tsx

```diff
@@ -161,7 +170,7 @@ const Menu = ({
   };
 
   return (
-    <ClickAwayListener onClickAway={onClose || null}>
+    <ClickAwayListener onClickAway={isButtonMenu ? null : onClose || null}>
```

Comment:
Suggestion; instead of calling it `isButtonMenu`, I think we might want a prop like `isClickAway` (default true)? That way it's clear what it's doing. As a user of `Menu` it's hard to tell what `isButtonMenu` does by looking at it